### PR TITLE
better configure storage class docs

### PR DESCRIPTION
**Pull request by nicksnyder**
some other random auto formatting happened too

### CODE REVIEW

File: docs/configure.md [hunk 1]
```diff
@@ -393,67 +393,66 @@ See [the official documentation](https://kubernetes.io/docs/concepts/configurati
...
-     - [Azure](https://kubernetes.io/docs/concepts/storage/storage-classes/#azure-disk):
+### Other cloud providers
 
-       ```yaml
-       # base/sourcegraph.StorageClass.yaml
-       provisioner: kubernetes.io/azure-disk
-       parameters:
-         storageaccounttype: Premium_LRS
-       ```
+The [the documentation](https://kubernetes.io/docs/concepts/storage/storage-classes/#parameters) for your cloud provider.
```
Comment (nicksnyder) [inline docs/configure.md]:
```suggestion
Read [the documentation](https://kubernetes.io/docs/concepts/storage/storage-classes/#parameters) for your cloud provider.
```

File: docs/configure.md [hunk 2]
```diff
@@ -393,67 +393,66 @@ See [the official documentation](https://kubernetes.io/docs/concepts/configurati
...
-       deploy: sourcegraph
-   #
-   # The values of the "provisioner" and "parameters" fields will differ depending on the cloud provider that you are using. Please read through https://kubernetes.io/docs/concepts/storage/storage-classes/ in order to know what values to add. 🚨 We recommend specifying SSDs as the disk type if possible. 🚨
-   #
-   # For example, if you are using GKE with a cluster whose nodes are all in the "us-central1-a" zone, you could use the following values:
-   #
-   # provisioner: kubernetes.io/gce-pd
-   # parameters:
-   #  type: pd-ssd
-   #  zones: us-central1-a
```
Comment (ggilmore) [inline docs/configure.md]: Do you know if this field is now unnecessary?

File: docs/configure.md [hunk 3]
```diff
@@ -393,67 +393,66 @@ See [the official documentation](https://kubernetes.io/docs/concepts/configurati
...
 
-Sourcegraph expects there to be storage class named `sourcegraph` that it uses for all its persistent volume claims. This storage class must be configured before applying the base configuration to your cluster. The configuration details differ depending on your hosting provider, so you should:
+Sourcegraph expects there to be storage class named `sourcegraph` that it uses for all its persistent volume claims. This storage class must be configured before applying the base configuration to your cluster.
 
-1. Create a stub `base/sourcegraph.StorageClass.yaml`.
+Create `base/sourcegraph.StorageClass.yaml` with the appropriate configuration for your cloud provider and commit the file to your fork.
 
-   ```yaml
-   # base/sourcegraph.StorageClass.yaml
-   kind: StorageClass
```
Comment (ggilmore) [inline docs/configure.md]: If you are going to delete this snippet, you should explicitly document that you need the `deploy: sourcegraph` label.

File: docs/configure.md [hunk 4]
```diff
@@ -393,67 +393,66 @@ See [the official documentation](https://kubernetes.io/docs/concepts/configurati
...
 
-     - [AWS](https://kubernetes.io/docs/concepts/storage/storage-classes/#aws):
+```yaml
+# base/sourcegraph.StorageClass.yaml
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: sourcegraph
+  labels:
+    deploy: sourcegraph
```
Comment (ggilmore) [inline docs/configure.md]: The Azure documentation notes that certain VMs might not be able to attach a premium LRS disk. This should be stated here as well.

File: docs/configure.md (hunk 2 above)
Comment (nicksnyder) [reply docs/configure.md]: Not necessary, and I tested this as part of 3.1 release testing

image

File: docs/configure.md (hunk 3 above)
Comment (nicksnyder) [reply docs/configure.md]: Moved to "Other cloud providers" section.

**Pull request merged**